.github/workflows/ruff_linter.yml

```diff
@@ -25,13 +25,18 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install ruff
+        pip install ruff==0.6.8
```

Comment:
how come?

Reply:
Rule set changes with versions so this is to avoid sudden breakage if they release a newer version

Reply:
@drisspg any preference on this?

Cause btw now locally vs remotely there'll be ruff deviations https://github.com/pytorch/ao/blob/main/dev-requirements.txt#L22

Reply:
That's fine; I think we just need to also pin in dev-requirments.txt